catroid/src/org/catrobat/catroid/content/Project.java

```diff
@@ -405,6 +419,17 @@ public Sprite getSpriteBySpriteName(Sprite searchedSprite) {
 		return spriteBySpriteName;
 	}
 
+	public Sprite getSpriteBySpriteName(String searchedSpriteName) {
```

Comment:
Thank you for the hint. I see that I don't even use the method, so I removed it. :smiley_cat: